packages/views/src/errorBoundary.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+
+export type FallbackRender = (errorData: {
+  error: Error;
+  errorInfo: React.ErrorInfo | null;
+  resetError(): void;
+}) => React.ReactElement;
```

Comment:
Can we use `ReactNode` as the return value instead?

Reply:
I think best would be `ReactChild`.